core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -212,7 +230,15 @@ protected void doStop() throws Exception {
 
     ServletContextHandler context = new ServletContextHandler(ServletContextHandler.SESSIONS);
     context.setContextPath("/");
-    context.addServlet(servletHolder, "/*");
+
+    ServletHolder defaultHolder = new ServletHolder("default", DefaultServlet.class);
+    defaultHolder.setInitParameter("dirAllowed", "false");
+
+    ResourceCollection staticResources = getStaticResources();
+    if(staticResources != null) {
```

Comment:
style nit -- should have a space after `if`. sorry we don't have checkstyle on this yet...